src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="/"
+        class="moyo__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo_logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
+              apple
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="#samsung"
+            >
+              samsung
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="#smartphones"
+            >
+              smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              data-qa="hover"
+              href="#laptops_&_computers"
+            >
+              laptops & computers
+            </a>
```

Comment:
The `href` attribute in the `<a>` tag should not contain spaces or special characters that are not URL-encoded. Instead of `#laptops_&_computers`, consider using `#laptops-and-computers` or another valid URL fragment.